File: src/main/resources/static/scripts/model/ConferenceCall.js
class ConferenceCall {

    /**
     * @param {{callerUserId: Number, otherUserId: Number, roomNumber: Number}} properties
     */
    static fromProperties(properties) {
        return new ConferenceCall(properties.callerUserId, properties.otherUserId, properties.roomNumber);
    }

    /**
     * @param {Number} callerUserId
     * @param {Number} otherUserId
     * @param {Number} roomNumber
     */
    constructor(callerUserId, otherUserId, roomNumber) {
        this.callerUserId = callerUserId;
        this.otherUserId = otherUserId;
        this.roomNumber = roomNumber;
    }
}

export default ConferenceCall;

File: src/main/java/fr/marcworld/webrtcdemo/dtos/ConferenceCall.java
package fr.marcworld.webrtcdemo.dtos;

public class ConferenceCall {

    private int callerUserId;
    private int otherUserId;
    private int roomNumber;

    public ConferenceCall() {
    }

    public ConferenceCall(int callerUserId, int otherUserId, int roomNumber) {
        this.callerUserId = callerUserId;
        this.otherUserId = otherUserId;
        this.roomNumber = roomNumber;
    }

    public int getCallerUserId() {
        return callerUserId;
    }

    public void setCallerUserId(int callerUserId) {
        this.callerUserId = callerUserId;
    }

    public int getOtherUserId() {
        return otherUserId;
    }

    public void setOtherUserId(int otherUserId) {
        this.otherUserId = otherUserId;
    }

    public int getRoomNumber() {
        return roomNumber;
    }

    public void setRoomNumber(int roomNumber) {
        this.roomNumber = roomNumber;
    }
}
